ansible/roles/anitya-dev/files/librariesio-consumer.service

```diff
@@ -4,7 +4,7 @@ After=network.target
 
 [Service]
 Environment="ANITYA_WEB_CONFIG=/home/vagrant/anitya.toml"
-ExecStart=librariesio_consumer.py
+ExecStart=/home/vagrant/devel/anitya/librariesio_consumer.py
```

Comment:
You shouldn't need that the script should be in standard PATH, it is installed as binary

Reply:
yes, you are right, changed path to scrip, provided by binary install